### ISSUE TITLE
[.NET] selectAction title should be passed to UI automation for accessibility

### ISSUE THREAD

**Issue by andrewleader** [microsoft/AdaptiveCards#2426]
**Automatically generated task. Any updates to the title and body will be overwritten. Do NOT modify the labels.**

.NET implementation of #1298
